lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/cc/PrintHandler.java

```diff
@@ -1089,9 +1118,12 @@ public void checkCurriculum(Element the_xml) {
 
   public void setManifestXml(Element the_xml) {
       manifestXml = the_xml;
+      log.debug("manifest xml: "+the_xml);
+  }
 
-	  log.debug("manifest xml: "+the_xml);
-
+  public void setCanvasModuleMetaXml(Element the_xml) {
+      canvasModuleMeta = the_xml;
+      log.debug("canvas_meta_module xml: "+the_xml);
```

Comment:
I would remove the log.debug within these set methods. Also line 1121

Reply:
This is a pretty important moment - it signals that the Canvas extension processing is enabled.   Important moment for debugging.